src/@orb.yml

```diff
@@ -3,10 +3,9 @@ version: 2.1
 description: >
   Test and deploy APEX applications to Salesforce.
 
-# This information will be displayed in the orb registry and is not mandatory.
 display:
-  home_url: "https://www.website.com/docs"
-  source_url: "https://www.github.com/EXAMPLE_ORG/EXAMPLE_PROJECT"
+  source_url: "https://developer.salesforce.com"
 
 orbs:
-  sfdx-cli: circleci/salesforce-sfdx@1.0.0
\ No newline at end of file
+  # This is temporary while the sfdx Orb is being worked on.
+  sfdx: circleci/salesforce-sfdx@dev:alpha
```

Comment:
When will this be updated?

Reply:
As soon as https://github.com/CircleCI-Public/Salesforce-sfdx-cli-orb/pull/13 is merged.